controller/rpc/pb/controller.proto

```diff
@@ -23,6 +23,11 @@ service ControllerService {
     rpc ReplicaPrepareRebuild(ReplicaAddress) returns (ReplicaPrepareRebuildReply);
     rpc ReplicaVerifyRebuild(ReplicaAddress) returns (ControllerReplica);
 
+    rpc BackupReplicaMapCreate(BackupReplicaMap) returns (google.protobuf.Empty);
+    rpc ReplicaAddressGet(ReplicaAddressRequest) returns (ReplicaAddressResponse);
```

Comment:
`BackupReplicaMapGet`, change request and response name as well. Same above.

Please pay close attention to the naming pattern.